libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -609,43 +609,135 @@ $button-normal-active-shadow: transparentize($button-normal-active-border-color,
 }
 
 .gd-data-header-menu-section {
+    position: absolute;
     display: flex;
     flex: 1 0 auto;
 
     .icon-sources {
         border-bottom-right-radius: 0;
         border-top-right-radius: 0;
 
+        &.is-normal::before {
+            margin-top: 2px;
+            margin-left: 2px;
+            background: url("../../assets/data_source_normal_state.svg") no-repeat;
+        }
+
+        &.is-active::before {
+            background: url("../../assets/data_source_active_state.svg") no-repeat;
+        }
+
+        &.disabled::before {
+            background: url("../../assets/data_source_disabled_state.svg") no-repeat;
+        }
+
+        &::before {
+            top: 11px;
+            margin-top: 1.2px;
+
+            &.disabled {
+                color: #b0beca;
+            }
+
+            &.is-normal {
+                color: #6d7680;
+            }
+        }
+
         &:hover {
             border-color: $gd-color-highlight;
+            z-index: 100;
+
+            &.is-normal::before {
+                color: #6d7680;
+            }
+
+            &.disabled {
+                color: #b0beca;
+            }
+
+            &.is-normal {
+                color: #464e56;
+            }
         }
     }
 
     .icon-model {
         border-radius: 0;
+        left: 89px;
+
+        &.is-normal::before {
+            margin-top: 2px;
+            margin-left: 2px;
+            background: url("../../assets/model-normal-state.svg") no-repeat;
+        }
+
```

Comment:
yes we dont need it because the svg in goodstrap didn change. Source icon is change to the new icon so I created 3 new icons for it for model we just need one new icon